qml/components/StickerPicker.qml

```diff
@@ -54,31 +78,28 @@ Item {
         color: Theme.overlayBackgroundColor
         opacity: 0.7
     }
-
-    Flickable {
-        id: stickerPickerFlickable
+    SilicaListView {
+        id: stickerPickerListView
         anchors.fill: parent
-        anchors.margins: Theme.paddingMedium
-
-        contentHeight: stickerPickerColumn.height
         clip: true
 
-        Column {
-            id: stickerPickerColumn
-            spacing: Theme.paddingMedium
-            width: stickerPickerFlickable.width
+        model: stickerPickerOverlayItem.installedStickerSets
+
+        header: Column {
+            width: stickerPickerListView.width
+            height: childrenRect.height
```

Comment:
This sort of binding looks strange for a Column. Doesn't it automatically set its own height to the total height of its children, laid out vertically?

Reply:
That's what I thought, yes. 
But it does not, for some reason unknown to me. 
If height isn't defined, we only get the height of the Label.

Reply:
I have a feeling that it's somehow related to the `visible` attribute. Would it make sense to do `height: (count > 0) ? Theme.itemSizeExtraLarge : 0` instead? That would even allow to animate `recentStickersGridView` height, if needed.

Reply:
It would make more sense to do a mix of that and the existing code for the whole column. This would also hide the Label if there are no recent stickers. 
Alternatively, display another Label describing there's nothing to see here.
I don't think a transition/animation is needed here, the recently used stickers don't really update "in place".

```
Column {
 visible: recentStickersGridView.visible
 height: visible? childrenRect.height : 0
 […]
}
```
or something in that regard. Thoughts?

Reply:
I agree that we don't need an animation here. But the childrenRect also looks strange to me. I changed it to use a calculated height for all children including padding if the count is greater than zero - along with some other optimizations.